includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1761,17 +1761,14 @@ public static function get_reader_friendly_error_type_text( $error_type ) {
 	 * @return string The label.
 	 */
 	public static function get_details_summary_label( $validation_error ) {
-		if (
-			AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_ATTR === $validation_error['code'] ||
-			AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_TAG === $validation_error['code'] ||
-			AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_PROCESSING_INSTRUCTION === $validation_error['code'] ||
-			AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_PROPERTY_IN_ATTR_VALUE === $validation_error['code'] ||
-			AMP_Tag_And_Attribute_Sanitizer::MISSING_MANDATORY_PROPERTY === $validation_error['code'] ||
-			AMP_Tag_And_Attribute_Sanitizer::MISSING_REQUIRED_PROPERTY_VALUE === $validation_error['code'] ||
-			'duplicate_element' === $validation_error['code']
-		) {
+		$error_type = isset( $validation_error['type'] ) ? $validation_error['type'] : null;
+		$node_type  = isset( $validation_error['node_type'] ) ? $validation_error['node_type'] : null;
+
+		if ( self::CSS_ERROR_TYPE === $error_type ) {
+			$summary_label = sprintf( '<%s>', $validation_error['node_name'] );
+		} elseif ( isset( $validation_error['parent_name'] ) ) {
```

Comment:
Nice, it looks like this is where `<amp-img>` or `<img>` is assigned as the `parent_name`.